docs/UserGuide.adoc

```diff
@@ -199,70 +199,81 @@ along with their index which can be used to display actual consultation details.
 
 
 === Medicine Management
-[[addMed, Add medicine]]
-==== Add medicine into storage: `addMed`
+[[adddirec, Add directory]]
+==== Add new directory into storage: `adddirec`
 
```

Comment:
follow the UG structure we have agreed upon:

preface
diagram number
parameters
example
results